nixos/modules/services/misc/paperless.nix

```diff
@@ -439,5 +489,37 @@ in
         gid = config.ids.gids.paperless;
       };
     };
-  };
+  }) (lib.mkIf (cfg.enable && cfg.exporter.enable) {
+
+    systemd.tmpfiles.rules = [
+      "d '${cfg.exporter.directory}' - ${cfg.user} ${config.users.users.${cfg.user}.group} - -"
+    ];
+
+    systemd.timers.paperless-exporter = lib.mkIf (cfg.exporter.onCalendar != null) {
+      description = "paperless exporter timer";
+      wantedBy = [ "timers.target" ];
+      timerConfig = {
+        OnCalendar = cfg.exporter.onCalendar;
+        AccuracySec = "5m";
+      };
+    };
+
+    systemd.services.paperless-exporter = {
+      description = "paperless exporter service";
+      serviceConfig.User = cfg.user;
+      script = ''
+        set -e
+        cd ${cfg.dataDir}
```

Comment:
Nit: [`WorkingDirectory=`](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#WorkingDirectory=)

Reply:
Sure. However, this makes it easier to manually run the script for debugging purposes.